website/content/SDK/keys/_index.md

```diff
@@ -1,6 +1,64 @@
 ---
-title: keys
-keywords: ""
-excerpt: ""
+title: API Keys
+keywords: "api,keys,key,secret,token,bearer,credentials,secrets,data,sources"
+excerpt: "This guide explains how to manage API keys within an OpenBB SDK environment."
 geekdocCollapseSection: true
 ---
+The `keys` module provides the same functionality as the <a href="https://openbb-finance.github.io/OpenBBTerminal/#accessing-other-sources-of-data-via-api-keys" target="_blank"> Keys menu</a> does in the <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/" target="_blank">OpenBB Terminal application</a>. Each data source requiring authentication will have slight differences of required inputs. Refer to the table <a href="https://openbb-finance.github.io/OpenBBTerminal/#accessing-other-sources-of-data-via-api-keys" target="_blank">here</a> for a compiled list of links for each data source.
+
+![openbb.keys.mykeys()](https://user-images.githubusercontent.com/85772166/198151758-7fdf1e00-29c2-4fb1-bfb1-7421cc9b8986.png)
+
+## How to Use
+
+### Expected values for all sources is printed with:
```

Comment:
I disagree with the headers. This is the actual description and not a title. Let's change these to normal paragraph text. I'd also advise to use these in longer sentences explaining more about what the user is looking at precisely. For example:

_To access Python's built-in help functionality, use `help(openb.keys.reddit)`. With this, you will be able to understand how the functionality needs to be used. Alternatively, you can also find this documentation [here](https://openbb-finance.github.io/OpenBBTerminal/SDK/keys/reddit/)._